ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -17,9 +16,11 @@ import scala.util.Try
 
 object Cli {
 
-  private def reportUsageOfDeprecatedOption[B](
-      option: String
-  ) = { (_: Any, config: B) =>
+  private val Name = "ledger-api-test-tool"
+
+  private[this] implicit val tlsVersionRead: Read[File] = Read.reads(Paths.get(_).toFile)
```

Comment:
Shouldn't this have a more generic name? It's used for various parameters.

Reply:
Ok